auth-method-oauth2.js

```diff
@@ -1871,7 +1872,7 @@ class AuthMethodOauth2 extends AmfHelperMixin(AuthMethodBase) {
               @input="${this._valueHandler}"
               autocomplete="on"
               data-persistent="true"
-              hiddable
+              ?hiddable="${!showClientFields}"
               data-visible="implicit authorization_code"
```

Comment:
Instead of conditioning `hiddable` use `data-visible` property and add `client_credentials`.

Reply:
Take a look into this styles definition: https://github.com/advanced-rest-client/auth-methods/blob/stage/auth-method-oauth2.js#L269

Reply:
Added `client_credentials` to `data-visible` and revert hiddable changes